recipes/manif/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "cci.20240804":
```

```suggestion
  "cci.20240804":
```

For new recipes, we're going with the `0.0.2.cci.20240804` scheme, even if it's not supported in Conan 2. I wonder why they stopped releasing new versions and whether it would be feasible to have a new one 🤔

---

Hey there, `manif` author here. I'm planning on a new, long over due, release asap. I'm simply getting some features to land before that.